.github/workflows/survey-ui-test.yml

```
@@ -6,7 +6,8 @@ on:
   push:
     branches: [ main ]
   # We are not running on Pull Request, since we need the Google API Key
-
+  pull_request:
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)